animator/src/lib.rs

```diff
@@ -348,17 +308,19 @@ impl ControllerBuilder {
 
     pub fn http_lights(mut self, path: &str) -> Result<Self, Box<dyn Error>> {
         info!("Using http light client with endpoint {}", path);
-        self.client_builder = self
-            .client_builder
-            .with(Box::new(client::http::HttpLightClient::new(path)));
+        self.client_builder = self.client_builder.with(Box::new(
+            client::http::HttpLightClient::new(path).with_backoff(),
+        ));
         Ok(self)
     }
 
     pub fn tcp_lights(mut self, path: &str) -> Result<Self, Box<dyn Error>> {
         info!("Using tcp light client with endpoint {}", path);
-        self.client_builder = self
-            .client_builder
-            .with(Box::new(client::tcp::TcpLightClient::new(path)));
+        self.client_builder = self.client_builder.with(Box::new(
+            client::tcp::TcpLightClient::new(path)
+                .with_backoff()
+                .with_start_delay(Duration::milliseconds(125)),
```

Comment:
nit: maybe add a comment how this magic value was chosen?

Reply:
![XKCD Randomness](https://github.com/mrozycki/rustmas/assets/509418/b20d6f50-f211-4f96-8a07-aae6ddb49afe)